.travis.yml

```diff
@@ -13,5 +13,8 @@ before_script:
 after_script:
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
 
+env:
+  global:
+    - CC_TEST_REPORTER_ID=859fcfe88b00c026d15dce30e838e2299face8088b49fe62bc3a02d1507ce3d5
```

Comment:
I don't believe this is necessary to expose here. It's already available in the Travis env vars: https://travis-ci.org/sul-dlss/sdr-client/settings Do you think we should prefer to keep this in a public config file? I don't believe it's very sensitive, but I also don't think we need it in two places. Cc: @sul-dlss/infrastructure-team since this potentially impacts current practice across our repos.

Reply:
It wasn't working before and so I copied the pattern from argo: https://github.com/sul-dlss/argo/blob/master/.travis.yml#L48

Reply:
@jcoyne Are you open to removing this from your branch and pushing the result to see if it's actually needed?